### DISCUSSION TITLE
Authentication changes?

### DISCUSSION THREAD

**Discussion by AndyMillne** [Art-of-WiFi/UniFi-API-client#146] [Q&A] [answered]
Have there been any recent changes to the UDM-Pro that makes this no longer work? I had this set up last year but coming back to it I'm struggling to get the API client to log in successfully.

I'm getting a 404 error when connecting to /api/login (ip masked in output)

Debug output:

-----------LOGIN-------------
Array
(
    [url] => https://***.***.***.***:8443/api/login
    [content_type] => 
    [http_code] => 404
    [header_size] => 73
    [request_size] => 255
    [filetime] => -1
    [ssl_verify_result] => 18
    [redirect_count] => 0
    [total_time] => 0.005488
    [namelookup_time] => 0.00026
    [connect_time] => 0.00026
    [pretransfer_time] => 0.000909
    [size_upload] => 59
    [size_download] => 0
    [speed_download] => 0
    [speed_upload] => 11800
    [download_content_length] => 0
    [upload_content_length] => 59
    [starttransfer_time] => 0.005067
    [redirect_time] => 0
    [redirect_url] => 
    [primary_ip] => 212.159.73.208
    [certinfo] => Array
        (
        )

    [primary_port] => 8443
    [local_ip] => 192.168.0.51
    [local_port] => 64245
    [http_version] => 2
    [protocol] => 2
    [ssl_verifyresult] => 0
    [scheme] => HTTPS
    [appconnect_time_us] => 260
    [connect_time_us] => 260
    [namelookup_time_us] => 260
    [pretransfer_time_us] => 909
    [redirect_time_us] => 0
    [starttransfer_time_us] => 5067
    [total_time_us] => 5488
)

----------RESPONSE-----------
HTTP/1.1 404 
Content-Length: 0
Date: Wed, 09 Feb 2022 14:46:19 GMT

-----------------------------

**Comment by AndyMillne** [comment, marked as answer]
Nevermind, I was including the port in the controller URL.

**Comment by malle-pietje** [comment]
The results of true rubber ducking this :wink:
